code-review/README.md

```diff
@@ -26,12 +26,14 @@ Everyone
 Having Your Code Reviewed
 -------------------------
 
+* A picture, or better, a gif (use <a href="http://www.cockos.com/licecap/">LiceCap</a>), goes a long way. Make your PR sexy.
+* If a picture does not work, a short explanation of what the PR does is necessary. Give some context and don't assume the reviewer knows the project from head to toe.
 * Be grateful for the reviewer's suggestions. ("Good call. I'll make that
   change.")
 * Don't take it personally. The review is of the code, not you.
 * Explain why the code exists. ("It's like that because of these reasons. Would
   it be more clear if I rename this class/file/method/variable?")
-* Extract some changes and refactorings into future tickets/stories.
+* Extract some changes and refactorings into future tickets/stories. As much as possible, a PR should either be a feature, or a technical PR, not both.
 * Link to the code review from the ticket/story. ("Ready for review:
   https://github.com/organization/project/pull/1")
 * Push commits based on earlier rounds of feedback as isolated commits to the
```

Comment:
Yes, that's true. I've tried to do it a bit sometimes, but I haven't done much programming to build a good habit of it. I think that commit squashing can be a nice thing to do and learning git is something good. =)

I'd propose having some kind of discussion/presentation on git practices and go from there into deciding.